api/scpca_portal/serializers.py

```diff
@@ -218,6 +218,18 @@ class Meta:
         )
 
 
+class DatasetCreateSerializer(serializers.ModelSerializer):
+    class Meta:
+        model = Dataset
+        fields = (
+            "id",
+            "data",
+            "email",
+            "format",
+        )
```

Comment:
Same comment as earlier PR. Expand this list manually for now and we will re-consolidate them after.